Address-Finder/Address-Finder/ViewController.swift

```diff
@@ -6,21 +6,148 @@
+        } else {
+            doSearchAddress(keyword: " ")
+        }
+    }
+    
+    func doSearchAddress(keyword: String) {
+        let headers: HTTPHeaders = [ "Authorization": "KakaoAK 754d4ea04671ab9d7e2add279d718b0e" ]
+        let parameters: [String: Any] = [ "query": keyword ]
+        
+        Alamofire.request("https://dapi.kakao.com/v2/local/search/address.json",
+                          method: .get,
+                          parameters: parameters,
+                          headers: headers).responseJSON{ (response) in
+            switch response.result {
+            case .success(let result):
+                    do {
+                        let jsonData = try JSONSerialization.data(withJSONObject: result, options: .prettyPrinted)
+                        let getInstanceData = try JSONDecoder().decode(APIResponse.self, from: jsonData)
+                        self.resultList = getInstanceData.documents
+                        print(self.resultList)
+                    }
+                    catch {
+                        print(error.localizedDescription)
+                    }
+                self.addressTableView.reloadData()
+            case .failure(let error):
+                print(error)
+            }
+        }
+    }
+}
+
+extension ViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return resultList.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let addressTableCell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as! AddressTableCell
+
+        if let roadAddress = resultList[indexPath.row].roadAddress?.addressName {
+            addressTableCell.roadAddressLabel.text = ("도로명: ") + roadAddress
+        }
+        else {
+            addressTableCell.roadAddressLabel.text = ("도로명: ")
+        }
+
+        if let jibeonAddress = resultList[indexPath.row].address?.addressName {
+            addressTableCell.jibeonAddressLabel.text = ("지번: ") + jibeonAddress
+        }
+        else {
+            addressTableCell.roadAddressLabel.text = ("지번: ")
+        }
         
-        let SearchController = UISearchController(searchResultsController: nil)
-        SearchController.searchBar.placeholder = "Search Address"
-        self.navigationItem.searchController = SearchController
+        return addressTableCell
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return view.frame.height / 10
     }
+}
 
+extension ViewController: UITableViewDelegate {
 
 }
 
+extension ViewController: UITextFieldDelegate {
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        if let keyword = addressSearchTextField.text {
+            doSearchAddress(keyword: keyword)
+        } else {
+            doSearchAddress(keyword: " ")
+        }
```

Comment:
```suggestion
        if let keyword = textField.text {
            doSearchAddress(keyword: keyword)
        } else {
            doSearchAddress(keyword: " ")
        }
```
이게 더 맞을듯 싶어요!! 바로 값을 가져와도 되지만 delegate 를 쓰는 이유가 퇴색되기 때문에 

해당 메서드가 제공하는 textField를 사용하는게 더 맞을 것 같아요.

```swift
// 방법1
if let keyword = textField.text {
    doSearchAddress(keyword: keyword)
} else {
    doSearchAddress(keyword: " ")
}

// 방법 2
let keyword = textField.text ?? ""
doSearchAddress(keyword: keyword)
```
이런 식으로도 사용 가능하니 참고하시면 됩니다